src/metabase/legacy_mbql/schema.cljc

```diff
@@ -1554,6 +1554,9 @@
    {:error/message "Distinct, non-empty sequence of Field clauses"}
    (helpers/distinct [:sequential {:min 1} Field])])
 
+(mr/def ::OrderBys
```

Comment:
I toyed with the idea of making this stricter in legacy as well, eventually decided against it but we should be able to update this schema without having to go update code like `driver.common.table-rows-sample` above.